### PR TITLE
Add perimeter subscriber

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 1]
```diff
@@ -89,6 +94,28 @@ public function addWorkflowProfile(GroupFormEvent $event)
...
+            $workflowProfileCollections = $group->getWorkflowProfileCollections();
+            $perimeter = new Perimeter();
+            $perimeter->setType(ContentTypeInterface::ENTITY_TYPE);
+
+            foreach ($workflowProfileCollections as $key => $workflowProfile) {
+                if ($key != NodeInterface::ENTITY_TYPE && count($workflowProfile->getProfiles()) > 0) {
+                    $perimeter->addItem($key);
+                }
+            }
+
```
Comment (ngilain) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: @itkg-nanne : I'm not sure at all about this perimeter ? What is saved here ? Is it for the following use case : Contributor on News and Validator on Cars ?

**Comment by itkg-nanne** [comment]
update done @ngilain

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 2]
```diff
@@ -89,6 +94,8 @@ public function addWorkflowProfile(GroupFormEvent $event)
             'sub_group_id' => 'backoffice',
         ));
         $builder->get('workflow_profile_collections')->addModelTransformer($this->workflowProfileCollectionTransformer);
+        $builder->addEventSubscriber($this);
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: ?? i don't understand this

**Comment by itkg-nanne** [comment]
update done @alavieille

File: WorkflowAdminBundle/Tests/EventSubscriber/AddGroupWorkflowProfileSubscriberTest.php [hunk 3]
```diff
@@ -6,6 +6,7 @@
 use OpenOrchestra\BaseBundle\Tests\AbstractTest\AbstractBaseTestCase;
 use Phake;
 use OpenOrchestra\GroupBundle\GroupFormEvents;
+use Symfony\Component\Form\FormEvents;
```
Comment (ngilain) [inline WorkflowAdminBundle/Tests/EventSubscriber/AddGroupWorkflowProfileSubscriberTest.php]: @itkg-nanne : is this use required as nothing has changed in the code ?

**Comment by itkg-nanne** [comment]
update done @ngilain

**Pull request merged**